pyscriptjs/src/utils.ts

```diff
@@ -111,3 +111,16 @@ export function joinPaths(parts: string[], separator = '/') {
     }
     return res;
 }
+
+export function createDeprecationWarning(msg: string, elementName: string): void {
+    const banners = document.getElementsByClassName('alert-banner py-warning');
+    let bannerCount = 0;
+    for (const banner of banners) {
+        if (banner.innerHTML.includes(elementName)) {
+            bannerCount++;
+        }
+    }
+    if (bannerCount == 0) {
+        _createAlertBanner(msg, "warning");
+    }
+}
```

Comment:
So this relies on `innerHTML` to include the elementName to detect duplicates, but doesn't necessarily _force_ the message to include the elementName. I wonder if something the following would be useful, so that we don't have to count on future deprecation-message-writers to include the elementName in their message?

```js
_createAlertBanner(`DeprecationWarning (${elementName}): ${msg}`, "warning");
```

Reply:
That could be a nice approach, although I think that `createDeprecationWarning` will only exist until we remove these elements since it's likely that other deprecations will be a bit different. 

The current deprecation message mentions the deprecated element and what users should use instead.  

Are you happy to have a short-lived function like this one?

Reply:
Makes sense to me!